invokeTarkovTextract/lambda_function.py

```diff
@@ -0,0 +1,21 @@
+from urllib.parse import unquote_plus
+
+import boto3
+
+s3_client = boto3.client('s3')
+textract_client = boto3.client('textract')
+
+SNS_TOPIC_ARN = 'arn:aws:sns:eu-west-1:832044451952:AmazonTextract-TarkovTopic'    # We need to create this
+ROLE_ARN = 'arn:aws:iam::832044451952:role/TextractServiceRole'   # This role is managed by AWS
```

Comment:
```suggestion
ROLE_ARN = 'arn:aws:iam::832044451952:role/TextractServiceRole'   # This role is managed by AWS
```

Should we be committing ARNs?